mod/thewire/actions/delete.php

```diff
@@ -9,7 +9,7 @@
 
 // Make sure we actually have permission to edit
 $thewire = get_entity($guid);
-if ($thewire->getSubtype() == "thewire" && $thewire->canEdit()) {
+if (elgg_instanceof($thewire, 'object', 'thewire') && $thewire->canEdit()) {
```

Comment:
Both. I meant general preference for instance checks.
On Apr 10, 2015 3:22 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In mod/thewire/actions/delete.php
> <https://github.com/Elgg/Elgg/pull/8143#discussion_r28143381>:
>
> > @@ -9,7 +9,7 @@
> >
> >  // Make sure we actually have permission to edit
> >  $thewire = get_entity($guid);
> > -if ($thewire->getSubtype() == "thewire" && $thewire->canEdit()) {
> > +if (elgg_instanceof($thewire, 'object', 'thewire') && $thewire->canEdit()) {
>
> Recycle views? Do you mean actions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8143/files#r28143381>.
>